layer demo/layer-demo.bas

```diff
@@ -117,3 +125,27 @@
 11180 DATA 76,28,144  : REM JMP $901C
 11190 DATA 88         : REM CLI
 11200 DATA 96         : REM RTS
+12000 REM
+12001 REM - MACHINE CODE FOR LAYER 1 QUICK FILL
```

Comment:
This is pretty much just a copy of the machine code above, but modified to fill with 0's.  There's probably a way to consolidate this, but this works well enough.